packages/block-library/src/post-comments/style.scss

```diff
@@ -45,12 +46,17 @@
 	}
 
 	.comment-meta {
+		font-size: 0.875em;
 		line-height: 1.5;
 		margin-left: -3.25em;
+
+		b {
+			font-weight: normal;
+		}
 	}
 
 	.comment-body .commentmetadata {
-		font-size: 0.75em;
+		font-size: 0.875em;
```

Comment:
This change unifies with the comment-meta on the top level comment, and the reply button, nice. That said, 0.75 both feels like simpler em math, but also felt visually better to me: 0.875 is smaller, but it doesn't look smaller enough to be _intentional_. This is not a strong feeling.

Reply:
Yeah, these numbers all seem a little random. 😅 The reason I went with the higher value of `0.875em` here is because it's _slightly_ larger, and I thought that would be a safer route. 

For example, if `1em` here were the typical browser default value of `16px`, then `0.875em` turns out to be `14px`, and `0.75em` is `12px`. `12px` seems _way_ too small for anything these days. `14px` is still small but better at least.